lib/eclipse/include/opm/parser/eclipse/EclipseState/AquiferCT.hpp

```diff
@@ -0,0 +1,155 @@
+            for (size_t aquctRecordIdx = 0; aquctRecordIdx < aquctKeyword.size(); ++ aquctRecordIdx) 
+            {
+                const auto& aquctRecord = aquctKeyword.getRecord(aquctRecordIdx);
+
+                aquctParams.at(aquctRecordIdx).aquiferID = aquctRecord.getItem("AQUIFER_ID").template get<int>(0);
+                aquctParams.at(aquctRecordIdx).h = aquctRecord.getItem("THICKNESS_AQ").getSIDouble(0);
+                aquctParams.at(aquctRecordIdx).phi_aq = aquctRecord.getItem("PORO_AQ").getSIDouble(0);
+                aquctParams.at(aquctRecordIdx).d0 = aquctRecord.getItem("DAT_DEPTH").getSIDouble(0);
+                aquctParams.at(aquctRecordIdx).C_t = aquctRecord.getItem("C_T").getSIDouble(0);
+                aquctParams.at(aquctRecordIdx).r_o = aquctRecord.getItem("RAD").getSIDouble(0);
+                aquctParams.at(aquctRecordIdx).k_a = aquctRecord.getItem("PERM_AQ").getSIDouble(0);
+                aquctParams.at(aquctRecordIdx).theta = aquctRecord.getItem("INFLUENCE_ANGLE").getSIDouble(0);
+                aquctParams.at(aquctRecordIdx).c1 = 0.008527; // We are using SI
+                aquctParams.at(aquctRecordIdx).c2 = 6.283;
+                aquctParams.at(aquctRecordIdx).inftableID = aquctRecord.getItem("TABLE_NUM_INFLUENCE_FN").template get<int>(0);
+                aquctParams.at(aquctRecordIdx).pvttableID = aquctRecord.getItem("TABLE_NUM_WATER_PRESS").template get<int>(0);
+
+                // std::cout << aquctParams.at(aquctRecordIdx).inftableID << std::endl;
+                // Get the correct influence table values
+                if (aquctParams.at(aquctRecordIdx).inftableID > 1)
+                {
+                    const auto& aqutabTable = eclState.getTableManager().getAqutabTables().getTable(aquctParams.at(aquctRecordIdx).inftableID - 1);
+                    const auto& aqutab_tdColumn = aqutabTable.getColumn(0);
+                    const auto& aqutab_piColumn = aqutabTable.getColumn(1);
+                    aquctParams.at(aquctRecordIdx).td = aqutab_tdColumn.vectorCopy();
+                    aquctParams.at(aquctRecordIdx).pi = aqutab_piColumn.vectorCopy();
+                }
+                else
+                {
+                    set_default_tables(aquctParams.at(aquctRecordIdx).td,aquctParams.at(aquctRecordIdx).pi);
+                }
+
+                // We determine the cell perforation here. (We must provide OPM-grid and simulator 
+                // with the relevant information of the global cell id). This is a hack for now to test functionality
+                int cellID = 10 + aquctRecordIdx;
+
+                aquctParams.at(aquctRecordIdx).cell_id.push_back(cellID);
+            }
+
+            return aquctParams;
+        }
+
+        inline void set_default_tables(std::vector<double>& td, std::vector<double>& pi)
+        {
+            std::vector<double> default_pressure_ = { 0.112, 0.229, 0.315, 0.376, 0.424, 0.469, 0.503, 0.564, 0.616, 0.659, 0.702, 0.735, 
```

Comment:
Here initialize a reference with a local variable; might be some C++ copying saves you - but it does not look good.